assets/constants.ts

```diff
@@ -676,6 +676,9 @@ export = {
   // Used to disable account removal until it is fully implemented.
   "ENABLE_ACCOUNT_DELETION": false,
 
+  // Used to disable account data export until it is fully implemented.
+  "ENABLE_ACCOUNT_EXPORT": false,
```

Comment:
Generally, we don't use a verb as the prefix for the variable. Maybe `ACCOUNT_EXPORT_ENABLED`

Reply:
Ah -- I noticed this but I think it's fine to leave it as-is for consistency with ENABLE_ACCOUNT_DELETION. Would that be OK @DubeySandeep?

Reply:
I changed this, but as a heads up if you look 2 lines above you'll see another variable named in a very similar fashion.

Reply:
Oh should I cancel my push @seanlip ?

Reply:
I was asking to change this as other developers start using this kinda variable name (With a reason that we already have this kinda constants) and then they have to do a follow up commit to fix it. I'm fine if it's okay to have this (For consistency reason).